packages/utils/node-resolver-rs/src/lib.rs

```diff
@@ -477,7 +477,7 @@ impl<'a> ResolveRequest<'a> {
           && self.resolver.flags.contains(Flags::EXPORTS)
         {
           // An internal package #import specifier.
-          let package = self.find_package(self.from.parent().unwrap())?;
+          let package = self.find_package(self.from.parent().unwrap_or_else(|| self.from))?;
```

Comment:
Any context on this one? This changes behaviour outside of the feature flag right?

Reply:
It should be pretty safe. It's mostly to pass some of the integration tests, but it is a legitimate failure when `self.from` is the root directory. We shouldn't fail when `from` is the root, as `self.from.parent()` will be `None` in that case.